google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -42,6 +42,13 @@ class InstanceAdminEmulator final
     google::protobuf::TextFormat::PrintToString(*request, &request_text);
     std::cout << __func__ << "() request=" << request_text << "\n";
 
+    constexpr int kMaxInstanceIdLength = 33;
```

Comment:
The project is "east const" and we prefer `auto`: s/constexpr int/auto constexpr/ here and below.